src/frontend/src/apis/teams.api.ts

```diff
@@ -18,3 +18,9 @@ export const getSingleTeam = (id: string) => {
     transformResponse: (data) => JSON.parse(data)
   });
 };
+
+export const setTeamMembers = (id: string, payload: any) => {
```

Comment:
let's rename the payload to be `userIds` and make it type `number[]`

then instead of `{ ...payload }` we can just do `{ userIds }